prime-router/docs/releases/2021-08-10-release-notes.md

```diff
@@ -0,0 +1,20 @@
+#  ReportStream Release Notes
+
+*August 3, 2021*
+
+## General useful links:
+
+- All Schemas are documented here:  [Link to detailed schema dictionaries](../schema_documentation)
+- The Hub API is documented here: [Hub OpenApi Spec](../openapi.yml)
+- [Click here for all release notes](../releases)
+
+## For this release
+
+### New `correctedTestId` column
+
+A new column has been added to allow for test results to amend/update previous results.   This feature has been added for those who were first working with Waters, and are now sending CSV data directly to ReportStream.  It works like this:
```

Comment:
This is the same design as we used for SimpleReport, now re-used for all senders under the 'direct' parent schema.